acceptance/image/image.go

```diff
@@ -68,13 +71,41 @@ const (
 	title               = "org.opencontainers.image.title"
 )
 
+type Signature struct {
+	KeyID       string            `json:"keyid"`
+	Signature   string            `json:"sig"`
+	Certificate string            `json:"certificate,omitempty"`
+	Chain       []string          `json:"chain,omitempty"`
+	Metadata    map[string]string `json:"metadata,omitempty"`
+}
+
 type imageState struct {
-	Attestations map[string]string
-	Images       map[string]string
-	Signatures   map[string]string
+	AttestationSignatures map[string]Signature
+	Attestations          map[string]string
+	Images                map[string]string
+	ImageSignatures       map[string]Signature
+	Signatures            map[string]string
```

Comment:
What's the difference between `Signatures` and `ImageSignatures` ?

Reply:
Added a comment

Reply:
I still don't understand the difference between those two :sweat_smile:

Reply:
> I still don't understand the difference between those two sweat_smile

Fair, I'll reword/refactor this in a followup, this has been open for a minute now...